core/domain/story_domain.py

```diff
@@ -177,26 +176,69 @@ def create_default_story_node(cls, node_id):
             StoryNode. The StoryNode domain object with default
             value.
         """
-        return cls(node_id, [], [], [], '', '')
+        return cls(node_id, [], [], [], '', None)
+
+
+class StoryContents(object):
+    """Domain object representing the story_contents dict."""
+
+    def __init__(self, story_nodes):
+        """Constructs a StoryContents domain object.
+
+        Args:
+            story_nodes: list(StoryNode). The list of story nodes that are part
+                of this story.
+        """
+        self.nodes = story_nodes
+
+    def to_dict(self):
+        """Returns a dict representing this StoryContents domain object.
+
+        Returns:
+            A dict, mapping all fields of StoryContents instance.
+        """
+        return {
+            'nodes': [
+                node.to_dict() for node in self.nodes
+            ]
+        }
+
+    @classmethod
+    def from_dict(cls, story_contents_dict):
+        """Return a StoryContents domain object from a dict.
+
+        Args:
+            story_contents_dict: dict. The dict representation of
+                StoryContents object.
+
+        Returns:
+            StoryContents. The corresponding StoryContents domain object.
+        """
+        story_contents = cls([
+            StoryNode.from_dict(story_node_dict)
+            for story_node_dict in story_contents_dict['nodes']
+        ])
+
+        return story_contents
 
 
 class Story(object):
     """Domain object for an Oppia Story."""
 
     def __init__(
             self, story_id, title, description, notes,
-            story_nodes, schema_version, language_code, version,
+            story_contents, schema_version, language_code, version,
             created_on=None, last_updated=None):
-        """Constructs a Question domain object.
+        """Constructs a Story domain object.
 
         Args:
             story_id: str. The unique ID of the story.
             title: str. The title of the story.
             description: str. The high level desscription of the story.
             notes: str. A set of notes, that describe the characters,
                 main storyline, and setting.
-            story_nodes: list(StoryNode). List of the story nodes present in
-                the story.
+            story_contents: StoryContents. The dict representing the contents
```

Comment:
Docstring seems confused. The type specified indicates a domain object; the description represents a dict. Which is it?

Reply:
Done, it was a StoryChanges object.